public/locales/en/accounts.json

```diff
@@ -37,5 +37,6 @@
   "smartContractDetails": "Smart Contract Details",
   "deployer": "Deployer",
   "label": "Label",
-  "codeId": "Code Id"
+  "codeId": "Code Id",
+  "block": "Block"
```

Comment:
I'd suggest changing line 41 to "instBlock": "Instantiated At Block" as it is more understandable.

Reply:
Fixed.